src/vehicle_gui.cpp

```diff
@@ -2736,7 +2736,9 @@ struct VehicleViewWindow : Window {
 		}
 		this->FinishInitNested(window_number);
 		this->owner = v->owner;
-		this->GetWidget<NWidgetViewport>(WID_VV_VIEWPORT)->InitializeViewport(this, this->window_number | (1 << 31), _vehicle_view_zoom_levels[v->type]);
+
+        auto zoom_level = ZoomLevel(_vehicle_view_zoom_levels[v->type] + _gui_zoom - ZOOM_LVL_OUT_4X); // adjust zoom level for GUI scaling (HiDPI)
+        this->GetWidget<NWidgetViewport>(WID_VV_VIEWPORT)->InitializeViewport(this, this->window_number | (1 << 31), zoom_level);
```

Comment:
On each of the two lines you changed, you need to indent with 2 tab characters instead of 8 spaces.

Reply:
Thanks! I did see your comment on the other PR, will adjust my editor and update the PR. Strange that the `.editorconfig` isn't taking effect in CLion?